config/default.yml

```diff
@@ -30,6 +30,10 @@ Metrics:
 
 Metrics/BlockLength:
   Max: 25
+  Exclude:
+    - spec/
+    - app/serializers/
+    - lib/tasks/
```

Comment:
This should be in `config/rails.yml` config file.

Reply:
I agree, but `rails.yml` seems to override `default.yml` on this exlusion list

Reply:
It does not override it: it merges it. Thanks to this configuration: https://github.com/alpinelab/codestyle/blob/bd9fd8d4ba2351b1d45941f586a4b9a9d73915f4/config/default.yml#L3-L5